--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -231,14 +231,7 @@
 set (LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}" CACHE PATH "Installation path for libraries")
 
 # Make a version file containing the current version from git.
-include (GetGitRevisionDescription)
-git_describe (VERSION "--tags")
-if ("x_${VERSION}" STREQUAL "x_GIT-NOTFOUND" OR "x_${VERSION}" STREQUAL "x_-128-NOTFOUND")
-  message (WARNING " - Install git to compile a production UPM!")
-  set (VERSION "v0.7.0-dirty")
-endif ()
+set (VERSION "v0.7.0")
 
 message (STATUS "UPM Version ${VERSION}")
 
